### DISCUSSION TITLE
3.9.5.81怎么获取图片

### DISCUSSION THREAD

**Discussion by ywx1216756599** [ttttupup/wxhelper#387] [General]
各位大佬 3.9.5.81怎么获取图片啊 没有看到图片对应的地址

**Comment by miloira** [comment]
> 各位大佬 3.9.5.81怎么获取图片啊 没有看到图片对应的地址

3.9.5.81有可用的注入工具吗？

**Comment by chengxiangtong** [comment]
应该是要调用downloadAttach得到dat文件，再解码
